src/components/ClimbState.js

```diff
@@ -0,0 +1,5 @@
+export const ClimbState = Object.freeze({
```

Comment:
see https://stackoverflow.com/questions/36795819/when-should-i-use-curly-braces-for-es6-import. do you think it'd be cleaner (assuming it works) to just do `export default Object.freeze({...})` here and then do `import ClimbState from './ClimbState.js' later? that seems to be more in line with what vue does -- i'm assuming that .vue files just produce a single default export, so the curly braces aren't needed later. that seems like it may be simpler since we're just producing a single export here.